PromoExchange/batman
# Show selected main image on slack notification

https://trello.com/c/gwX5vAAF/1427-be-able-to-select-image-of-the-variant-you-want-from-purchase-page

The goal of this PR is to make images somewhat "selectable" for the Gooten store (mainly product color variants). The objective is to show the selected image in the slack purchase channel upon purchase, *NOT* to actually tie colors, variants, images, and purchases together (long term solution).

#### Test Plan:
1. Navigate to Gooten CS on this PR
2. Select `T-Shirts` option
3. Select one of the mini pics below the main pic. Ensure the main pic switches to that of the smaller, selected one.
4. Continue through a purchase all the way to the end, including entering in payment info for Stripe (dummy data)
5. Ensure that the #px-orders-test slack channels gets sent a notification of the buyer, price, and image of the product
